### PR TITLE
Update Twitter component configuration

### PR DESCRIPTION
**Pull request by klaasnicolaas**
**Description:**
Update style of Twitter component documentation to follow new configuration variables description.
Related to #6385.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/notify.twitter.markdown [hunk 1]
```diff
@@ -30,13 +30,33 @@ notify:
...
+access_token_secret:
+  description: Your "Access Token Secret" for the application.
+  required: true
+  type: string
+username:
+  description: Twitter handle without `@` or with `@` and quoting for direct messaging.
+  required: false
+  type: string
+{% endconfiguration %}
 - **name** (*Optional*): Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`.
```
Comment (tjorim) [inline source/_components/notify.twitter.markdown]: I think you forgot to remove this line.

**Review by tjorim — CHANGES_REQUESTED** [review]

File: source/_components/notify.twitter.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_components/notify.twitter.markdown]: Yep 😞

**Comment by klaasnicolaas** [comment]
Funny. Deployment fails but no error 😢

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**